src/ironhide/core.cljc

```diff
@@ -256,6 +264,26 @@
     path
     (into prefix path)))
 
+(defn- selectable?
+  "Returns true when ironhide path exists in context (ctx)
+   and false when ironhide path not exists in context (ctx).
+   Examples:
+     (selectable? {:b {nil 1}} [:b nil :cc]) => false
+     (selectable? {:b {nil 2}} [:b nil])     => true
+     (selectable? {:b {nil 2}} [:b :c])      => false
+     (selectable? {:a nil} [:a])             => true
+     (selectable? {} [:a])                   => false
+     (selectable? {:b {nil nil}} [:b nil])   => true
+     (selectable? {:a [[:v1] [:v2 :v3 :v4]]} [:a [:*] [:*]])    => true
+     (selectable? {:a [[:v1] [:v2 :v3 :v4]]} [:a [:*] [:*] :w]) => false
+   "
+  [ctx path]
+  (nil? (some #(nil? %)
+            (sp/select TREE-VALUES
+                       (sp/select
+                        (path->sp-path path)
+                        (sp/transform TREE-VALUES #(if (nil? %) :placeholder %) ctx))))))
```

Comment:
traversing whole data structure and updating all nil values will kill performance.